Replaced all split() call by preg_split()

arcadien opened the pull request.

xemle commented:
Thanks for the patch. What is the motivation to replace split() by preg_split()?

arcadien commented:
Hello,

Ubuntu's PHP current version is PHP 7 which no longer implements "plain" 
split(). So i updated the code to use preg_split everywhere.

Regards,

Aurelien


Le 23/12/2016 à 15:12, Xemle a écrit :
>
> Thanks for the patch. What is the motivation to replace split() by 
> preg_split()?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/xemle/phtagr/pull/15#issuecomment-268995556>, or 
> mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AByN19j-CmR32wgREgedfSKy0yF6u8lAks5rK9a4gaJpZM4LUzz_>.
>
